src/main/java/com/spartronics4915/frc2019/subsystems/CargoChute.java

```diff
@@ -95,52 +111,50 @@ public void onLoop(double timestamp)
                 SystemState newState = defaultStateTransfer();
                 switch (mSystemState)
                 {
-                    case RAMPING:
+                    case INTAKING:
```

Comment:
I'm not really sure why you've removed the transition between intaking and holding when the ball gets in position. I don't think it's appropriate to remove this automation.

I know you were concerned about shooting the ball out the top and fluctuating between holding and intaking, but as far as I'm concerned that just meant we should re-tune the IR sensor value. Removing the entire feature is not the right way to solve this problem, IMO.